### PR TITLE
gerbera: Update to 1.5.0.

### PR DESCRIPTION
**Pull request by mamins1376**
This introduces `libupnp1.12` as a way to gradually replace with `libupnp1.8`.

### CODE REVIEW

**Comment by mamins1376** [comment]
I already updated `common/shlibs` and the new package should have nothing to do with reverse dependencies of `libupnp1.8`. Why is `99-pkglint` making errors?

**Comment by mamins1376** [comment]
https://travis-ci.org/github/void-linux/void-packages/jobs/682466026#L4642

File: common/shlibs [hunk 1]
```diff
@@ -3087,6 +3087,9 @@ libmirage.so.11 libmirage-3.1.0_1
 libwkhtmltox.so.0 libwkhtmltopdf-0.12.5_1
 libixml.so.10 libupnp1.8-1.8.2_1
 libupnp.so.13 libupnp1.8-1.8.4_1
+libixml.so.11 libupnp1.12-1.12.1_1
+libupnp.so.11 libupnp1.12-1.12.1_1
```
Comment (Chocimier) [inline common/shlibs:3091]: This seems to be problem.

**Comment by Chocimier** [comment]
All packages listed there need to be revbumped and tested.

**Pull request closed without merging**